packages/plugin-ext/src/main/node/plugins-key-value-storage.ts

```diff
@@ -101,7 +101,7 @@ export class PluginsKeyValueStorage {
             return {};
         }
         try {
-            return await fs.readJSON(pathToFile);
+            return fs.readJSONSync(pathToFile);
```

Comment:
we should avoid to use sync method, since it blocks the backend.

Reply:
Do you think we can replace it with `fs.read` and `fs.write` to have it async but creating proper not empty content?

Reply:
Yeah, just looking now, how to avoid sync but still keep right file content.

Reply:
Found the core of the issue
https://stackoverflow.com/questions/30886217/node-js-fs-writefile-empties-the-file
so, I've changed it to save asynchronously to a temporary file (just suffix .tmp), and rename to the original file. And it solved the issue. Wanted to add a test, but did not manage to reproduce the issue there.